rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SQSRestServerBuilder.scala

```diff
@@ -141,7 +142,8 @@ case class TheSQSRestServerBuilder(providedActorSystem: Option[ActorSystem],
         changeMessageVisibilityBatch(p) ~
         deleteQueue(p) ~
         getQueueAttributes(p) ~
-        setQueueAttributes(p)
+        setQueueAttributes(p) ~
+        addPermissions(p)
```

Comment:
you are totally right about this but even if i change this to addPermission the errors stay the same:
> [error] /home/marcel/dev/sources/elasticmq-server/rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SQSRestServerBuilder.scala:116: not found: type AddPermissionDirectives
> [error]       with AddPermissionDirectives
> [error]            ^
> [error] /home/marcel/dev/sources/elasticmq-server/rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SQSRestServerBuilder.scala:146: not found: value addPermission
> [error]         addPermission(p)
> [error]         ^
> [error] two errors found
> [error] (elasticmq-rest-sqs/compile:compileIncremental) Compilation failed